ssl/statem/statem_dtls.c

```diff
@@ -1103,36 +1127,36 @@ int dtls1_get_queue_priority(unsigned short seq, int is_ccs)
      * Finished, it also maintains the order of the index (important for
      * priority queues) and fits in the unsigned short variable.
      */
-    return seq * 2 - is_ccs;
+    int lsb = (record_type == SSL3_RT_CHANGE_CIPHER_SPEC ? 1 : 0);
```

Comment:
Not sure whether the ternary operator is needed here as C is pretty clear on that results of logical operators are 0 or 1.

Also missing empty line after declarations.

Reply:
Good point, thanks!